api/bases/cinder.openstack.org_cinders.yaml

```diff
@@ -15,7 +15,16 @@ spec:
     singular: cinder
   scope: Namespaced
   versions:
-  - name: v1beta1
+  - additionalPrinterColumns:
+    - description: Status
+      jsonPath: .status.conditions[0].status
```

Comment:
I see this is consistent with the other operators (e.g. glance/keystone): as long as it's the pattern used by all the operators, lgtm.